### PR TITLE
Fxa browser link

### PR DESCRIPTION
**Pull request by mbirghan**
This PR adds a link to the FxA support form to the support flow error page. 
I also made changes to the standard error page to accommodate a secondary action button that is not a SignOut button. This can also be used for other error pages. 

In doing this I also moved the `VPN.logout()` action from the users of the `VPNSignOut` button to the button itself since I think this makes more sense. It would not make sense to have a `VPNSignOut` button that does a different action.

### CODE REVIEW

**Review by bakulf — APPROVED** [review]
Code-wise it's good! Also a nice cleanup. But I would wait for @lesleyjanenorton for the QML code review.

File: src/ui/views/ViewErrorFullScreen.qml [hunk 1]
```diff
@@ -16,10 +16,16 @@ VPNFlickable {
...
-    property var buttonObjectName
-    property var buttonOnClick
-    property var signOffLinkVisible: false
+
+    property var primaryButtonText
+    property var primaryButtonObjectName
+    property var primaryButtonOnClick
+
+    property var secondaryButtonText: ""
+    property var secondaryButtonObjctName
```
Comment (lesleyjanenorton) [inline src/ui/views/ViewErrorFullScreen.qml:25]: nit: missing an an e in 'Object'

**Review by lesleyjanenorton — COMMENTED** [review]
Looks good! I think we just need to add the string for `VPNl18n.InAppSupportWorkflowFxALinkText` to strings.yaml.

File: src/ui/views/ViewContactUs.qml [hunk 2]
```diff
@@ -58,9 +63,13 @@ Item {
...
                         headlineText: VPNl18n.InAppSupportWorkflowSupportErrorHeader,
                         errorMessage: VPNl18n.InAppSupportWorkflowSupportErrorText,
-                        buttonText: VPNl18n.InAppSupportWorkflowSupportErrorButton,
-                        buttonOnClick: contactUsRoot.tryAgain,
-                        buttonObjectName: "errorTryAgainButton"
+                        primaryButtonText: VPNl18n.InAppSupportWorkflowSupportErrorButton,
+                        primaryButtonOnClick: contactUsRoot.tryAgain,
+                        primaryButtonObjectName: "errorTryAgainButton",
+                        secondaryButtonIsSignOff: false,
+                        secondaryButtonText: VPNl18n.InAppSupportWorkflowFxALinkText,
```
[the remark this answers is not in the document]
Comment (mbirghan) [reply src/ui/views/ViewContactUs.qml:70]: Yes, I will talk to @chrisrooneys about adding this sting.

**Review by lesleyjanenorton — APPROVED** [review]

**Comment by lesleyjanenorton** [comment]
@mbirghan anything I can do to help with the string here?

**Comment by mbirghan** [comment]
No I will add the string today and then we can land it

**Review by flodolo — APPROVED** [review]

**Pull request merged**